README.md

```diff
@@ -1,22 +1,58 @@
 # MultiplayerSample Project
 
-A simple third-person multiplayer sample for O3DE.
+A simple third-person multiplayer sample for O3DE. Its lazer tag in space!
```

Comment:
"laser". Well technically L.A.S.E.R.
Light 
Amplification by 
Stimulated 
Emission of 
Radiation